src/services/zipRecruiter.jsx

```diff
@@ -0,0 +1,18 @@
+const fetchData = async () => {
+    try {
+      const url = "https://api.ziprecruiter.com/partner/v0/job/emlwcmVjcnVpdGVy-3215";
+      const response = await fetch(url);
+      if (!response.ok) {
+        throw new Error(
+          `Failed to fetch data: ${response.status} ${response.statusText}`
+        );
+      }
+      const result = await response.json();
+      console.log(result);
+      return result;
+    } catch (error) {
+      console.error(error);
+    }
+  };
+
+  export {fetchData};
```

Comment:
Github doesn't like it when you don't end files with a newline, so you should always end any file with one. Read more here: https://stackoverflow.com/a/45393212/20419934